test/lib/mayaUsd/render/mayaToHydra/cpp/testMayaUsdUfeItems.cpp

```diff
@@ -35,7 +35,7 @@ bool IsUfeLight(const HdSceneIndexBasePtr& sceneIndex, const SdfPath& primPath)
     }
     // The shape prim is used to display the light's wireframe and is the only prim we want to allow
     // for UFE lights
-    bool isShapePrim = primPath.GetElementString().find("ufeLightProxyShape") < std::string::npos;
+    bool isShapePrim = primPath.GetElementString().find("ufeLightProxy") < std::string::npos;
```

Comment:
This seems wrong : if a prim comes in named ufeLightProxy, this method will now return false when it should return true. The original bug that this test looks for is that previously, MayaUSD added its lights both through UFE and through its UsdStageSceneIndex, which led to the light being present twice. The fix I did was to filter out the UFE light (ufeLightProxy), but not its wireframe display (ufeLightProxyShape). So conceptually, this method returns primPath == "ufeLightProxy" (I can't recall if there was a reason I didn't do this directly and instead used find calls). So if this test fails to me this points to a different issue.